packages/msgpck-repr/msgpck-repr.1.5/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+name: "msgpck-repr"
+maintainer: "Vincent Bernardoff <vb@luminar.eu.org>"
+authors: "Vincent Bernardoff <vb@luminar.eu.org>"
+homepage: "https://github.com/vbmithr/ocaml-msgpck"
+license: "ISC"
+dev-repo: "git+https://github.com/vbmithr/ocaml-msgpck.git"
+bug-reports: "https://github.com/vbmithr/ocaml-msgpck/issues"
+depends: [
+  "dune" {build & >= "1.11.4"}
```

Comment:
```suggestion
  "dune" {>= "1.11.4"}
```

Dune should not be tagged as build dependency (https://github.com/ocaml/opam-repository/issues/14185)